src/Tools/dotnet-format/dotnet-format.csproj

```diff
@@ -0,0 +1,68 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>Microsoft.CodeAnalysis.Tools.CodeFormatter</RootNamespace>
+    <Description>Command line tool for formatting code files.</Description>
+
+    <IsPackable>true</IsPackable>
+    <PackAsTool>true</PackAsTool>
+    <!-- These identifiers are for generating the shim'd core executables for signing. Not all options
+         from $(RoslynPortableRuntimeIdentifiers) work or make sense in this context. -->
+    <PackAsToolShimRuntimeIdentifiers>win-x64;win-x86;osx-x64</PackAsToolShimRuntimeIdentifiers>
+    <!-- Redirect so that executables can be signed before packaging. -->
+    <PackagedShimOutputRootDirectory>$(OutputPath)Exes\$(MSBuildProjectName)\</PackagedShimOutputRootDirectory>
```

Comment:
Where does the output go if you don't specify this?

Reply:
The shims were building to the `$(OutputPath)Obj\$(MSBuildProjectName)\` folder.

Reply:
And they are never copied to the Exe path?